src/cli/test_command.zig

```diff
@@ -246,6 +252,19 @@ pub const CommandLineReporter = struct {
         this.summary.expectations += expectations;
         this.jest.tests.items(.status)[id] = TestRunner.Test.Status.todo;
     }
+
+    pub fn printSummary(this: *CommandLineReporter, summary: Summary, start_time: i128, total_tests: ?u32) void {
```

Comment:
Since `start_time` is always `bun.start_time`, this doesn't need to be a function argument.